be/src/service/starrocks_main.cpp

```diff
@@ -303,18 +304,10 @@ int main(int argc, char** argv) {
     options.conf_path = string(getenv("STARROCKS_HOME")) + "/conf/";
     starrocks::StorageEngine* engine = nullptr;
 
-    if (without_storage) {
-        auto st = starrocks::DummyStorageEngine::open(options, &engine);
-        if (!st.ok()) {
-            LOG(FATAL) << "fail to open StorageEngine, res=" << st.get_error_msg();
-            exit(-1);
-        }
-    } else {
-        auto st = starrocks::StorageEngine::open(options, &engine);
-        if (!st.ok()) {
-            LOG(FATAL) << "fail to open StorageEngine, res=" << st.get_error_msg();
-            exit(-1);
-        }
+    auto st = starrocks::StorageEngine::open(options, &engine);
```

Comment:
one more thing, cn uses DummyStorageEngine, who uses options.conf_path to initialize the `ClusterIdMgr`. BE storageEngine will write cluster id to each available DataDir.